library/general/allowedrepos/suite.yaml

```diff
@@ -0,0 +1,13 @@
+kind: Suite
+apiVersion: test.gatekeeper.sh/v1alpha1
+tests:
+- name: allowed-repos
+  template: template.yaml
+  constraint: samples/repo-must-be-openpolicyagent/constraint.yaml
+  cases:
+  - name: example-allowed
+    object: samples/repo-must-be-openpolicyagent/example_allowed.yaml
```

Comment:
Question: if there's no assertion, does it default to `violations: no`? if so, when does a user want to explicitly use assertions with `violation: no`?

Reply:
I think this thread is most relevant:

https://github.com/open-policy-agent/gatekeeper/pull/1519#discussion_r697701073

Agreed that zero assertions => implied assertion that the object will pass validation.

IMO it should be that zero assertions => zero tests against the object => the test can never fail

Reply:
If we leave the behavior as-is, we risk winding up with weird edge cases if the tests become more complex (e.g. imagine we are not testing validation)

Reply:
wdyt @willbeason ?

Reply:
I made the change you suggested.

Reply:
thanks! but does that mean gator doesnt require assertions?

Reply:
Will told me today he wanted to change it so that a test with no assertions is always invalid and will fail.